### PR TITLE
ODIM-3208 add rpc handler for add/remove actions of aggregate

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: svc-aggregation/agmodel/system.go [hunk 1]
```diff
@@ -709,3 +709,56 @@ func GetAllKeysFromTable(table string) ([]string, error) {
...
+
+//AddElementsToAggregate add elements to the aggregate
+func AddElementsToAggregate(aggregate Aggregate, aggregateURL string) *errors.Error {
+	agg, err := GetAggregate(aggregateURL)
+	if err != nil {
+		return err
+	}
+	aggregate.Elements = append(aggregate.Elements, agg.Elements...)
+
+	if err := DeleteAggregate(aggregateURL); err != nil {
```
Comment (jeevan-kamkar) [inline svc-aggregation/agmodel/system.go]: why there is a delete and create there can be new function which can update existing aggregate  2 db calls can be avoided

File: svc-aggregation/agmodel/system.go [hunk 2]
```diff
@@ -709,3 +709,56 @@ func GetAllKeysFromTable(table string) ([]string, error) {
...
+}
+
+//RemoveElementsFromAggregate remove elements from an aggregate
+func RemoveElementsFromAggregate(aggregate Aggregate, aggregateURL string) *errors.Error {
+	agg, err := GetAggregate(aggregateURL)
+	if err != nil {
+		return err
+	}
+	aggregate.Elements = removeElements(aggregate.Elements, agg.Elements)
+	if err := DeleteAggregate(aggregateURL); err != nil {
```
Comment (jeevan-kamkar) [inline svc-aggregation/agmodel/system.go]: why there is a delete and create there can be new function which can update existing aggregate 2 db calls can be avoided

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

File: svc-aggregation/agmodel/system.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-aggregation/agmodel/system.go]: updated

File: svc-aggregation/agmodel/system.go (hunk 2 above)
Comment (irappa-pattar) [reply svc-aggregation/agmodel/system.go]: updated

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

File: svc-aggregation/agmodel/system_test.go [hunk 3]
```diff
@@ -1084,3 +1084,77 @@ func TestDeleteAggregate(t *testing.T) {
 	err = DeleteAggregate(aggregateURI)
 	assert.NotNil(t, err, "err should not be nil")
 }
+
```
Comment (Bharath-KKB) [inline svc-aggregation/agmodel/system_test.go:1087]: why are these test codes getting added now ?

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
why are we adding test cases now and not before development?

File: svc-aggregation/agmodel/system_test.go (hunk 3 above)
Comment (irappa-pattar) [reply svc-aggregation/agmodel/system_test.go:1087]: these are model test cases added now, will add these also before development next time

**Review by Bharath-KKB — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]
LGTM

**Pull request merged**